Sources/CocoaLumberjack/DDFileLogger.m

```diff
@@ -947,7 +957,12 @@ - (void)lt_maybeRollLogFileDueToSize {
     // We specifically wrote our own getter/setter method to allow us to do this (for performance reasons).
 
     if (_maximumFileSize > 0) {
-        unsigned long long fileSize = [_currentLogFileHandle offsetInFile];
+        unsigned long long fileSize;
+        if (@available(macOS 10.15, iOS 13.0, tvOS 13.0, watchOS 6.0, *)) {
+            [_currentLogFileHandle getOffset:&fileSize error:nil];
```

Comment:
Again, if they give us API with errors, why we should ignore it?

Reply:
Yes, we shouldn't ignore it!